src/apps/annotation-image/ImageAnnotationDesktop.tsx

```diff
@@ -87,7 +89,7 @@ export const ImageAnnotationDesktop = (props: ImageAnnotationProps) => {
 
   const [leftPanelOpen, setLeftPanelOpen] = useState(false);
 
-  const [rightPanelOpen, setRightPanelOpen] = useState(false);
+  const { rightPanelOpen, rightPanelTab, setRightPanelOpen } = useAnnotationsView();
```

Comment:
I think that's a good hook to have - with one caveat below...

(BTW, I might lean towards renaming it `useAnnotationViewState` or `useAnnotationViewUIState`. Not sure what you think?)